tests/integration/generate_fixtures/go_ethereum.py

```diff
@@ -139,22 +137,23 @@ def generate_go_ethereum_fixture(destination_dir):
         keyfile_path = os.path.join(keystore_dir, common.KEYFILE_FILENAME)
         with open(keyfile_path, "w") as keyfile:
             keyfile.write(common.KEYFILE_DATA)
+        keyfile_pw = os.path.join(keystore_dir, common.KEYFILE_PW_TXT)
+        with open(keyfile_pw, "w") as keyfile_pw_file:
+            keyfile_pw_file.write(common.KEYFILE_PW)
         genesis_file_path = os.path.join(datadir, "genesis.json")
         with open(genesis_file_path, "w") as genesis_file:
             genesis_file.write(json.dumps(common.GENESIS_DATA))
 
-        geth_ipc_path_dir = stack.enter_context(common.tempdir())
-        geth_ipc_path = os.path.join(geth_ipc_path_dir, "geth.ipc")
-
+        geth_ipc_path = f"{datadir}/geth.ipc"
         geth_port = get_open_port()
         geth_binary = common.get_geth_binary()
 
         with get_geth_process(
             geth_binary=geth_binary,
             datadir=datadir,
             genesis_file_path=genesis_file_path,
-            geth_ipc_path=geth_ipc_path,
             geth_port=geth_port,
+            keyfile_pw=keyfile_pw,
```

Comment:
I thought we got rid of passing this around in `get_geth_process`. We either can leave this (nbd) or, since we know where it lives, we can do what we do in the other instances where we spin up geth: `os.path.join(datadir, "keystore", "pw.txt")`. Take or leave, I just remember removing this at some point.